### PR TITLE
Add an operative manager role

### PR DESCRIPTION
**Pull request by pixeltrix**

### CODE REVIEW

File: RepairsApi/V2/Controllers/OperativesController.cs [hunk 1]
```diff
@@ -56,8 +56,8 @@ public async Task<IActionResult> GetOperative([FromRoute][Required] string opera
         [Produces("application/json")]
         [ProducesResponseType(typeof(List<OperativeRequest>), 200)]
         [ProducesDefaultResponseType]
-        [Authorize(Roles = UserGroups.Agent + "," + UserGroups.ContractManager + "," + UserGroups.AuthorisationManager)]
-        public async Task<IActionResult> ListOperatives(OperativeRequest operativeRequest)
+        [Authorize(Roles = UserGroups.OperativeManager)]
```
Comment (Asura5) [inline RepairsApi/V2/Controllers/OperativesController.cs:59]: Just confirming that this endpoint is not used by existing users, and if it is, that these users have been given this role?

Comment (pixeltrix) [reply RepairsApi/V2/Controllers/OperativesController.cs:59]: No, it's still only deployed to the development environment - roles were copy/pasted from another endpoint

File: RepairsApi/Dockerfile [hunk 2]
```diff
@@ -3,8 +3,10 @@ FROM mcr.microsoft.com/dotnet/core/sdk:3.1
 WORKDIR /app
 
 # Copy csproj and restore as distinct layers
-COPY *.csproj ./
-RUN dotnet restore
+COPY ./RepairsApi.sln ./
```
Comment (kylealexanderhackney) [inline RepairsApi/Dockerfile:6]: Are these changes necessary?

Comment (pixeltrix) [reply RepairsApi/Dockerfile:6]: To be able to run `make serve` using Docker on a Mac, yes. Otherwise you have to run it through Visual Studio.

**Review by alex-demetriou — APPROVED** [review]
